pkgs/development/libraries/rubberband/default.nix

```
@@ -1,20 +1,28 @@
 { stdenv, fetchurl, pkgconfig, libsamplerate, libsndfile, fftw
 , vamp-plugin-sdk, ladspaH }:
 
-stdenv.mkDerivation {
-  name = "rubberband-1.8.1";
+stdenv.mkDerivation rec {
+  pname = "rubberband";
+  version = "1.8.2";
 
   src = fetchurl {
-    url = "http://code.breakfastquay.com/attachments/download/23/rubberband-1.8.1.tar.bz2";
-    sha256 = "0x9bm2nqd6w2f35w2sqcp7h5z34i4w7mdg53m0vzjhffnnq6637z";
+    url = "https://breakfastquay.com/files/releases/${pname}-${version}.tar.bz2";
+    sha256 = "1jn3ys16g4rz8j3yyj5np589lly0zhs3dr9asd0l9dhmf5mx1gl6";
   };
 
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ libsamplerate libsndfile fftw vamp-plugin-sdk ladspaH ];
 
+  # https://github.com/breakfastquay/rubberband/issues/17
+  postPatch = ''
+    substituteInPlace Makefile.in --replace \
+    'cp -f $(JNI_TARGET) $(DESTDIR)$(INSTALL_LIBDIR)/$(JNINAME)$(DYNAMIC_EXTENSION)' \
+    'test -f $(JNI_TARGET) && cp -f $(JNI_TARGET) $(DESTDIR)$(INSTALL_LIBDIR)/$(JNINAME)$(DYNAMIC_EXTENSION) || true'
+  '';
+
```

do you mind doing a fetchpatch on https://github.com/breakfastquay/rubberband/commit/419a9bcf7066473b0d31e9a8a81fe0b2a8e41fed 
```suggestion
  # https://github.com/breakfastquay/rubberband/issues/17
  # In master, but there hasn't been an official release
  patches = [
    (fetchpatch {
      url = "https://github.com/breakfastquay/rubberband/commit/419a9bcf7066473b0d31e9a8a81fe0b2a8e41fed.patch";
      sha256 = ...;
   })
```